tslint.json

```diff
@@ -0,0 +1,12 @@
+{
+	"defaultSeverity": "error",
+	"extends": [
+		"tslint:recommended",
+		"tslint-react",
+		"tslint-eslint-rules",
+		"tslint-config-prettier"
+	],
+	"jsRules": {},
+	"rules": {},
```

Comment:
"rules": {
    "prettier": true
  },

Reply:
There is probably no such rule:
https://palantir.github.io/tslint/rules/